lib/yaml_loader.cpp

```diff
@@ -40,7 +40,7 @@ struct RymlCallbackInitializer {
 
 static nlohmann::ordered_json ryml_to_nlohmann_json(const c4::yml::NodeRef& ryml_node) {
     if (ryml_node.empty()) {
-        return nullptr;
+        return nlohmann::ordered_json::object();
```

Comment:
@a-w50 
I found the issue with testing with a ['broken' config](https://github.com/EVerest/everest-framework/blob/fix-unit-tests/tests/test_configs/broken_config_config.yaml), but it also seems to occur with empty config files as you said.

The error
```
[json.exception.type_error.306] cannot use value() with null
```
occured in [this line](https://github.com/EVerest/everest-framework/blob/9604e06820ea671b68d097870d2b80b6c99bfa1e/lib/runtime.cpp#L136).

Since there is a default value given it should'nt brake on empty objects.

With the mentioned fix it doesn't brake. Seems to make a difference to return nullptr or empty json here..


_In case of broken config (invalid yaml) it shouldn't return anything, but throw a exception, but I guess the 'broken' config I use is not broken enough_